### PR TITLE
Update and rename Nirmala_Ramnarine_BCO_000537.json to Updated_Nirmal…

### PR DESCRIPTION
**Pull request by nramnarine**
…a_Ramnarine_BCO_000537.json

Ready for Final Review

### CODE REVIEW

**Comment by jpat1546** [comment]
@nramnarine Reassigning back to you; please add the appropriate tags to this pull request. See [assignment details.](https://github.com/biocompute-objects/GW-SMHS-BIOC6223/blob/master/docs/5_ModifyBCO.md) Once they are added, reassign me and  I will review your final BCO.Thanks.

**Comment by nramnarine** [comment]
Okay, I placed the correct tags. Thank you Janisha!

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+                    "createdBy"
+                ],
+                "email": "nramnarine13@gwu.edu"
+            },
+            {
+                "name": "David T. Ting",
+                "orcid": "https://orcid.org/0000-0002-3261-2322",
+                "affiliation": "Massachusetts General Hospital Cancer Center, MASSACHUSETTS GENERAL HOSPITAL CANCER CENTER, Dana-Farber/Harvard Cancer Center, Massachusetts General Hospital, Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": " Ben S. Wittner",
+                "orcid": "https://orcid.org/0000-0001-5657-9849",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Matteo Ligorio",
+                "orcid": "https://orcid.org/0000-0001-8865-0881",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Health Sciences, University of Genoa, 16126 Genoa, Italy.",
+                "contribution": [
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json:62]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Nicole Vincent Jordan",
+                "orcid": "https://orcid.org/0000-0002-0137-5782",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 5]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Ajay M. Shah",
+                "orcid": "",
+                "affiliation": " Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 6]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "David T. Miyamoto",
+                "orcid": "https://orcid.org/0000-0003-3692-8823",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Radiation Oncology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 7]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nicola Aceto",
+                "orcid": "https://orcid.org/0000-0001-9579-6918",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 8]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": " Francesca Bersani",
+                "orcid": "https://orcid.org/0000-0002-0343-7078",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 9]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Brian W. Brannigan",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 10]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Kristina Xega",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 11]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Jordan C. Ciciliano",
+                "orcid": "https://orcid.org/0000-0003-1427-2608",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 12]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": " Huili Zhu",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 13]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Olivia C. MacKenzie",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 14]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Julie Trautwein",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 15]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Kshitij S. Arora",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 16]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mohammad Shahid",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 17]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Haley L. Ellis",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 18]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Na Qu",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 19]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nabeel Bardeesy",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 20]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Miguel N. Rivera",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 21]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Vikram Deshpande",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Pathology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 22]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Cristina R. Ferrone",
+                "orcid": "https://orcid.org/0000-0002-9465-7008",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 23]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": " Ravi Kapur",
+                "orcid": "",
+                "affiliation": " Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 24]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Sridhar Ramaswamy",
+                "orcid": "https://orcid.org/0000-0001-6715-4686",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 25]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": " Toshi Shioda",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 26]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mehmet Toner",
+                "orcid": "",
+                "affiliation": "Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 27]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Shyamala Maheswaran",
+                "orcid": "https://orcid.org/0000-0002-9356-1709",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA. Electronic address: maheswaran@helix.mgh.harvard.edu.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 28]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            },
+            {
+                "name": "Daniel A. Haber",
+                "orcid": "https://orcid.org/0000-0003-1378-6551",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA; Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA. Electronic address: haber@helix.mgh.harvard.edu.",
+                "contribution": [
+                    "authoredBy"
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112

File: Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json [hunk 29]
```diff
@@ -0,0 +1,563 @@
...
+                ],
+                "email": ""
+            }
+        ]
+    },
+    "usability_domain": [
+        "To identify the expression of extracellular matrix genes in mouse pancreatic CTCs and human CTCs originating from the pancreas, breasts, and prostate.",
+        "PMID: 25242334 PMCID: PMC4230325 DOI: 10.1016/j.celrep.2014.08.029",
```
Comment (jpat1546) [inline Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json]: Great job including the publication. I would just add some context to make this a complete sentence : "This BCO was adapted by [title][PMID/PMCID/DOI]

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@nramnarine 

I've reviewed your updated BCO and made comments on what further needs to be fixed (minor changes). 

Please make all the changes in the BCO Editor, download, and update this pull request file with the new JSON (copy/paste- same process as before). Once all changes have been made, assign this pull request back to me and I will review again. Until then, assigning the pull request back to you.

Great job with those modifications!

**Review by jpat1546 — APPROVED** [review]
### Your BCO is approved
Great job fixing all your assigned issues, @nramnarine .

Prior to submitting to the pFDA challenge, please make the following 2 changes to your BCO in the BCO Editor (then download and submit to the pFDA):
In Provenance domain: 
- Change reviewer status from "in review" to "approved"
- Change reviewer_comment from "Reviewed by BioCompute technical assistant, approval pending modification by author" to "Reviewed and approved by BioCompute technical assistant"

Leave me a comment if these instructions are not clear. Thanks!

**Pull request merged**